hooks/post_gen_project.py

```diff
@@ -26,10 +26,24 @@
 {% endif %}
 
 
+{% if cookiecutter.project_layout == "src" -%}
```

Comment:
I think you changed the value and conditions
so if cookiecutter.project_layout == "src", USE_SRC should be True
also change the if inside update_project_layout